sdk/avs/arm-avs/CHANGELOG.md

```diff
@@ -1,4 +1,92 @@
 # Release History
+
+## 7.1.0 (2025-10-16)
+
+### Features Added
+  - Added operation group LicensesOperations
+  - Added operation group MaintenancesOperations
+  - Added operation group ServiceComponentsOperations
```

Comment:
This was marked as internal in the TypeSpec. Is there a way to prevent this from showing up in the SDK?